src/index.html

```diff
@@ -1,13 +1,291 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Nothing</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href = "./images/logo.svg"
```

Comment:
Change the href value to './images/logo.png' as the logo.svg file does not exist.